### PR TITLE
network: fix quoting in bridgescript invocation

### PR DESCRIPTION
**Pull request by frobware**
The deviceNames to bridge were being quoted as "[ens4 ens5]", when
they should have been just "ens4 ens5". This commit simplifies the
arguments passed to the invocation of the bridgescript.

### CODE REVIEW

**Comment by macgreagoir** [comment]
LGTM. The `Sprintf` argument list is getting long, so a potential `text/template` candidate? I don't find it especially hard to follow, though.

**Comment by frobware** [comment]
I need to EOD but will follow-up with unit tests.

**Comment by frobware** [comment]
!!build!!

**Comment by macgreagoir** [comment]
Just commenting to ack the new comment explaining the use of WallClock. Cheers!

**Comment by frobware** [comment]
$$merge$$
